### ISSUE TITLE
[Fluxstore 1.7.6] Search function is not working

### ISSUE THREAD

**Issue by netdreamland** [inspireui/support#4314]
having downloaded and setup latest version 1.7.6 Fluxstore, having tried both search function on the top of the screen and tab bar, it's all nothing responding (the loading bar just keeps running)
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-07-22 at 17 27 50](https://user-images.githubusercontent.com/12180247/88160056-bda67f00-cc40-11ea-856f-8178da7a737c.png)

**Comment by minhcasi** [comment]
Thank you for the feedback.
Please help us to submit this form and fill all required info to continue with the support - http://verify.inspireui.com/ - please don't share the purchase code or screenshot public here as another user will use it for activation.

We will close this ticket in 24 hours if it is not provided enough detail, kindly reopen the ticket after you have done the form submission. Thank you for your time!

**Comment by netdreamland** [comment]
hi @minhcasi thanks for your response, I am not the first time to leave the question here, how come it still needs to do the verification everytime? it's so much confusing and clumsy.

By the way, I did finish the verification thru google form.

**Comment by minhcasi** [comment]
sorry for incontinence, as you may post the ticket from 4-5 months ago so that it was clean up by our robot. We got your request and sent it to the team for further investigating - thank you for your time!

**Comment by FaizalAndro** [comment]
Hi @netdreamland, In lib/models/filer_attribute.dart change this code: https://tppr.me/h3VbK

**Comment by minhcasi** [comment]
@netdreamland - could you try this solution to see if work fine on your site, or you could use the latest code on Gitlab - thank you!

**Comment by netdreamland** [comment]
hi @FaizalAndro it does not work , the loading icon still moving without change
@minhcasi I am using the latest version (1.7.6) already as told

**Comment by netdreamland** [comment]
having checked the script on lib/service/opencart.dart , the function has not been implemented, when could we have this function ready?
```
@override
  Future<List<FilterAttribute>> getFilterAttributes() {
    // TO-DO: implement getFilterAttributes
    throw UnimplementedError();
  }

```

**Comment by FaizalAndro** [comment]
@netdreamland Hi download new source code from codecanyon. Hope it helps, I checked and its working fine.

**Comment by minhcasi** [comment]
@netdreamland - could I confirm you are using Opencart on your website?

**Comment by netdreamland** [comment]
@minhcasi yes, I am using Opencart 3.0

**Comment by anmolmajhail** [comment]
> Hi @netdreamland, In lib/models/filer_attribute.dart change this code: https://tppr.me/h3VbK

This fixes the search issue in the WooCommerce version app.

**Comment by minhtruong315** [comment]
Hi @netdreamland You can try to replace getFilterAttributes function in services/opencart.dart like this
```
@override
  Future<List<FilterAttribute>> getFilterAttributes() async {
    return [];
  }
```

**Comment by netdreamland** [comment]
So it means the function in opencart is not yet implemented? If yes, when
is the time of completion?

It would be strange that if user search with empty result with all keywords

「minhtruong315 <notifications@github.com>」在 2020年8月3日 週一，10:57 寫道：

**Comment by minhtruong315** [comment]
Hi @netdreamland That function only supports for woo commerce. opencart doesn't support that function.

**Comment by netdreamland** [comment]
so how can the app user do a product search for the opencart's case? if it will never implement, i think it's hard to recommend for opencart's user in the future

**Comment by minhtruong315** [comment]
You can search product for the opencart. did you try to test search?

**Comment by netdreamland** [comment]
I have just tried the search function for the top banner, it's just the same.
I wonder if we have any code needed to change?
![Simulator Screen Shot - iPhone 11 Pro Max - 2020-08-03 at 16 14 16](https://user-images.githubusercontent.com/12180247/89161036-a18dd080-d5a4-11ea-8de2-c36f7db68982.png)

**Comment by minhtruong315** [comment]
@netdreamland can u send me full your source code via email: minhtruong315@gmail.com

**Comment by minhtruong315** [comment]
Hi, I'm closing this ticket as there is no response after more than 2 days, if you have any questions or reproducing the issue kindly reopen or create a new ticket - thank you!

**Comment by netdreamland** [comment]
i think i have mixed some code of 1.7.6 and patched 1.7.6 , i will log a new ticket once i faced the problem again.
Thanks very much!
